app/abtests/Test.scala

```diff
@@ -0,0 +1,54 @@
+package abtests
+
+import com.github.slugify._
+import play.api.libs.json.{JsValue, Json, Writes}
+import play.api.mvc.Cookie
+
+case class Test(name: String, audienceSize: Double, audienceOffset: Double, variants: Variant*) {
```

Comment:
given that the id generated is an int, would that make sense to have these as ints, and avoid the math below that may introduce rounding errors?